open-hackathon/src/hackathon/database/models.py

```diff
@@ -574,4 +574,119 @@ def json(self):
     def __repr__(self):
         return "VMConfig: " + self.json()
 
+# ------------------------------ Tables are introduced by azure-auto-deploy ------------------------------
+
+# ------------------------------ Tables for those logic around admin-site --------------------------------
+
+class AdminUser(Base):
+    __tablename__ = 'admin_user'
+    id = Column(Integer, primary_key=True)
+    name = Column(String(50))
+    nickname = Column(String(50))
+    openid = Column(String(100))
+    avatar_url = Column(String(200))
+    access_token = Column(String(100))
+    online = Column(Integer)  # 0:offline 1:online
+    create_time = Column(DateTime)
+    last_login_time = Column(DateTime)
+
+    def get_admin_user_id(self):
```

Comment:
fixed ！ and understand this function is needed when load user-info after login successfully